zerver/tests/test_hooks.py

```diff
@@ -6,6 +6,27 @@
 import ujson
 from six.moves import urllib
 
+class WebhookTestCase(AuthedTestCase):
+    STREAM_NAME = None
+    TEST_USER_EMAIL = None
```

Comment:
Might make sense to default TEST_USER_EMAIL to "hamlet" since it seems basically every test uses that anyway.